common/filesystem.py

```diff
@@ -42,6 +42,7 @@ def recreate_directory(directory, create_parents=True):
 
 def write(path, contents, open_flags='w'):
     """Opens file at |path| with |open_flags| and writes |contents| to it."""
+    # pylint: disable=unspecified-encoding
     with open(path, open_flags) as file_handle:
```

Comment:
Not sure if there is a better way to fix this:
We cannot simply add `encoding='utf-8'` as the `path` can be a binary file.

Reply:
OK, I left a todo on it for another PR as it is involved in several other functions.